partner_statement/views/activity_statement.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <template id="partner_statement.activity_statement_document">
+        <t t-call="web.external_layout">
+            <t t-set="o" t-value="o.with_context({'lang': lang})"/>
+            <t t-set="address">
+                <address t-esc="o" t-options='{"widget": "contact", "fields": ["address", "name"], "no_marker": True}' />
+                <div t-if="o.vat" class="mt16"><t t-esc="company.country_id.vat_label or 'Tax ID'"/>: <span t-field="o.partner_id.vat"/></div>
+            </t>
+            <div class="page">
+                <h2 >Statement of Account</h2>
+                <div id="informations" class="row mt32 mb32">
+                    <div class="col-3 bm-2">
+                        <strong>Date:</strong>
+                        <p t-esc="d['today']" class="m-0"/>
+                    </div>
+                    <div t-if="o.ref" class="col-3 bm-2">
+                        <strong>Partner Code:</strong>
+                        <p t-field="o.ref" class="m-0" />
+                    </div>
+                </div>
+
+                    <t t-if="d['currencies']">
+                        <br/>
+                        <t t-foreach="d['currencies'].items()" t-as="currency">
+                            <t t-set="display_currency" t-value="Currencies[currency[0]]" />
+                            <t t-set="currency" t-value="currency[1]" />
+                            <p>
+                                <span t-esc="'' if  account_type == 'receivable' else 'Supplier '"/>Statement between <span t-esc="d['start']" /> and <span t-esc="d['end']" /> in <span t-esc="display_currency.name"/>
```

Comment:
Supplier is not translatable

Reply:
Oh well, I'm done with this PR. Supplier statements isn't really a thing anyway. I only put it in for migrations, as far as I'm concerned they shouldn't be in there. The issue is in 11.0 just worse, so once this is merged, if someone fixes it we can forward it.